src/posts/postTransformers.js

```diff
@@ -0,0 +1,93 @@
+import posts from "./posts.json";
+
+const postsSorted = posts.sort((a, b) => (a.date < b.date ? 1 : -1));
+
+for (let post of postsSorted) {
+  post.slug = post.filename.substring(0, post.filename.indexOf("."));
+}
+
+const tags = postsSorted.map((post) => post.tags);
+const uniqueTags = [...new Set([].concat(...tags))].sort();
+
+const locationTags = uniqueTags.filter((tag) =>
```

Comment:
should we also use Canada as we are developing the model?

Reply:
In the location labels, you mean? This file is actually just a reimplementation of the old `Posts.jsx`, meaning it's live in the app since the code was merged today, so I'll open a new issue to add that. Any need for Nigeria, Israel, or others?